openapi.yaml

```diff
@@ -299,6 +299,13 @@ paths:
           schema:
             type: string
           example: Passed
+        - name: status
+          in: query
+          description: |
+            Execution Status for the run. Commonly queried values: 'building','generating','running'
```

Comment:
Can I have multiple of these status values in the query string ? or a comma-separated list of them set into one query parameter value ? or both ?

I would think "finished" was a common one, much more common that 'building' !